packages/nuxt/src/app/nuxt.ts

```diff
@@ -20,11 +20,16 @@ import type { LoadingIndicator } from '../app/composables/loading-indicator'
 import type { RouteAnnouncer } from '../app/composables/route-announcer'
 import type { ViewTransition } from './plugins/view-transitions.client'
 
-import type { NuxtAppLiterals } from '#app'
+import { type NuxtAppLiterals } from '#app'
+// @ts-expect-error virtual import
+import { buildId } from '#build/nuxt.config.mjs'
 
-const nuxtAppCtx = /* @__PURE__ */ getContext<NuxtApp>('nuxt-app', {
-  asyncContext: !!__NUXT_ASYNC_CONTEXT__ && import.meta.server,
-})
+/* @__NO_SIDE_EFFECTS__ */
```

Comment:
Here I'm not so sure about this, should I have kept `/* @__PURE__ */` or is `/* @__NO_SIDE_EFFECTS__ */` fine? I see it's rather used for functions.

Reply:
I think we can remove it now.

```suggestion
```